config/templates/cluster-deployment.yaml

```diff
@@ -81,6 +89,24 @@ objects:
   stringData:
     ssh-publickey: "${SSH_KEY}"
 
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: ClusterImageSet
+  metadata:
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: origin-latest
+  spec:
+    releaseImage: "registry.svc.ci.openshift.org/openshift/origin-release:v4.0"
```

Comment:
Does this work for you? I can't pull it.

Reply:
It did work in my manual test and I can see the imagestream on api.ci. Let me try it from a simple docker client

Reply:
I just spun a vm with just docker on it and I was able to pull it

Reply:
weird, i am getting unauthorized with both podman and docker.

Reply:
You may have an old token in your dockercfg for registry.svc.ci.openshift.org